toxcore/list.c

```diff
@@ -123,7 +123,7 @@ static bool resize(BS_List *list, uint32_t new_size)
 
     list->data = data;
 
-    int *ids = (int *)realloc(list->ids, sizeof(int) * new_size);
+    int *ids = (int *)realloc(list->ids, new_size * sizeof(int));
```

Comment:
why reorder?

Reply:
It's a tokstyle rule we used to apply but then it got lost because we moved to mem.

Reply:
hm, because everywhere else, that I could see from the small code review snippets in gh, it is `sizeof()*N`

Reply:
"Everywhere else" was 3 locations in the code, which I've fixed now. Let me know if you find any others.